bookkeeper-server/src/test/proto/TestDataFormats.proto

```diff
@@ -30,5 +30,5 @@ enum AuthMessageType {
  *
  */
 extend AuthMessage {
-    required AuthMessageType messageType = 1000;
+    optional AuthMessageType messageType = 1000;
```

Comment:
Why did you change this to "optional" ? anyway it is used only in tests there is no matter of compatibility with previous versions